idea/Hublsoft.Task.Web/Models/LoginMonitorModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Hublsoft.Web.Models
+{
+    public class LoginMonitorModel
+    {
+        public int Id { get; set; }
+        public string UserId { get; set; }
```

Comment:
Can you help me understand the discrepancy between `Email` which e.g. is used to label the field on the front-end, and `UserId` which is used in the back end? I may have missed some conversation about this.

Reply:
The thoughts were - UserId allows for other named identifiers to be used, maybe for the existing system which i believe does not have email as the identifier. However in this implementation it would be best to keep them the same.